addons/website_livechat/static/src/external_lib/livechat_button.js

```diff
@@ -0,0 +1,7 @@
+/** @odoo-module **/
+
+import LivechatButton from '@im_livechat/legacy/widgets/livechat_button';
+
+LivechatButton.include({
+    className: `${LivechatButton.prototype.className} o_bottom_fixed_element o_bottom_fixed_element_move_up o_website_livechat_button o-isExternalLib`,
```

Comment:
I don't understand why most of those classes are added in external lib when they only match style rules that are available in website?